src/main/java/picard/vcf/UpdateVcfSequenceDictionary.java

```diff
@@ -89,7 +91,15 @@ protected int doWork() {
         if (CREATE_INDEX)
             builder.setOption(Options.INDEX_ON_THE_FLY);
 
-        final VariantContextWriter vcfWriter = builder.setOutputFile(OUTPUT).build();
+        // handle stdout
+        if ( OUTPUT.getPath().equals(STD_OUT) ) {
```

Comment:
@yfarjoun The output cannot be redirected to a file via `System.setOut()`. the stream must be `System.out`, `builder.setOutputStream(System.out)`.